### ISSUE TITLE
Feature request-Dog vs Cat Classification using Transfer Learning

### ISSUE THREAD

**Issue by Shy029** [UppuluriKalyani/ML-Nexus#390]
I would like to propose the addition of a Dog vs Cat Image Classification System utilizing Transfer Learning to the project. This feature demonstrates the efficient reuse of pre-trained models for image classification tasks, helping contributors and users explore transfer learning concepts in deep learning with Python.

please assign this to me.

**Comment by Panchadip-128** [comment]
I have already created an advanced project on this topic Cat vs Dog Image classification using Transfer Learning algorithm. Please assign me this issue and i promise to deliver an exceptional and efficient ML model for the same . My test accuracy for the model has come around 97% without tested evidence of no overfitting . My model feature demonstrates the efficient reuse of pre-trained models for image classification tasks as well
![ct](https://github.com/user-attachments/assets/f5c73f5d-6f65-44a0-8f58-154b5f43442b)
![dg](https://github.com/user-attachments/assets/2b9afcb2-bbf4-447f-99ae-26cbd44a3a3e)
![results](https://github.com/user-attachments/assets/d8f2a41a-402c-4c50-9d74-5d57a4ca7391)

**Comment by Shy029** [comment]
I m working on it.
![Screenshot 2024-10-18 093507](https://github.com/user-attachments/assets/9d80bb75-55ce-4804-9be5-43c0e73c63f1)

**Comment by Shy029** [comment]
@UppuluriKalyani I have completed this one .As I had received a mail from github regarding this project .Kindly confirm it once.
